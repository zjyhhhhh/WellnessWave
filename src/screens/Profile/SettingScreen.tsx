import { View, Text } from "react-native";

const SettingScreen = ({}) => {
	return (
		<View>
			<Text>Setting Screen</Text>
		</View>
	);
};

export default SettingScreen;
