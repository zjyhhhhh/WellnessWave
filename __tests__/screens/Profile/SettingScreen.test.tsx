import "@testing-library/jest-dom";
import renderer from "react-test-renderer";
import SettingScreen from "../../../src/screens/Profile/SettingScreen";
import { render } from "@testing-library/react-native";

describe("Setting Screen", () => {
	it("renders correctly", () => {
		const tree = renderer.create(<SettingScreen />).toJSON();
		expect(tree).toMatchSnapshot();
	});

	it("displays the 'Setting Screen' text", () => {
		const { getByText } = render(<SettingScreen />);
		expect(getByText("Setting Screen")).toBeTruthy();
	});
});
