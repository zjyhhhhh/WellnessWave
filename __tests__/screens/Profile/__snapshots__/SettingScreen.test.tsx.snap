// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Setting Screen renders correctly 1`] = `
<View>
  <Text>
    Setting Screen
  </Text>
</View>
`;
